### PR TITLE
Allow batch operations for sql server

### PR DESCRIPTION
**Pull request by caiquecastro**

### CODE REVIEW

**Comment by koskimas** [comment]
So mssql returns an id for each inserted row, and not just the first or the last one?

**Comment by caiquecastro** [comment]
image

**Comment by caiquecastro** [comment]
image

File: lib/queryBuilder/operations/InsertOperation.js [hunk 1]
```diff
@@ -27,8 +27,8 @@ class InsertOperation extends QueryBuilderOperation {
   }
 
   async onBefore2(builder, result) {
-    if (this.models.length > 1 && !isPostgres(builder.knex())) {
-      throw new Error('batch insert only works with Postgresql');
+    if (this.models.length > 1 && !isPostgres(builder.knex()) && !isMsSql(builder.knex())) {
```
Comment (caiquecastro) [inline lib/queryBuilder/operations/InsertOperation.js]: What do you think about creating a function to wrap this logic. Like `supportsBatchInsert`

**Pull request merged**